resources/monitoring/values.yaml

```diff
@@ -23,12 +23,14 @@ global:
   alertTools:
     credentials:
       slack:
+        sendresolved: true
```

Comment:
please use camelCase: sendResolved as the key consists of two words

Reply:
changed now, thanks.